dbt_schema_builder/schema_builder.py

```diff
@@ -73,6 +74,7 @@ def parse_args(args):
         sys.exit(1)
 
     parsed = p.parse_args(args)
+    flags.set_from_args(parsed, {})
```

Comment:
What happens to `flags`?

Reply:
This was an issue with dbt cli command not picking the `--profile-dir` path, so this change is made to honor `--profile-dir` path

Reply:
I guess my question was more of: we are returning `parsed` but not `flags`. Are we leaving something out or does it create a side effect outside the scope of this function.